README.md

```diff
@@ -17,19 +17,137 @@ SPDX-License-Identifier: MIT
 [![](https://img.shields.io/endpoint?url=https%3A%2F%2Fswiftpackageindex.com%2Fapi%2Fpackages%2FStanfordSpezi%2FSpeziML%2Fbadge%3Ftype%3Dplatforms)](https://swiftpackageindex.com/StanfordSpezi/SpeziML)
 
 
-The Spezi machine learning module simplifies the integration to incorporate machine learning-related functionality in their Spezi-based applications.
+The Spezi ML Swift Package includes modules that are helpful to integrate ML-related functionality in your application.
```

Comment:
It would be amazing if we can show the UI components using screenshot in this PR, e.g., the two onboarding steps and the Chat view that we have implemented in the package. In the long run, this might need to move into a separate package if it is independent from Spezi ML but for now, it might be a great way to showcase this here 🎉

Reply:
Great idea - have added the 3 screenshots (two onboarding steps and chat view) in light and dark mode.